tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/ExternalSortExec.java

```diff
@@ -630,19 +636,43 @@ public TableStats getInputStats() {
     CLOSED
   }
 
+  private static class SimpleMerger extends PairWiseMerger {
```

Comment:
I think SimpleMerger is not proper name for this class. First time when I saw the next code, I can understand after checking the detail code of SimpleMerger class. How about "VectorComparePairWiseMerger".
'''
if (ComparableVector.isApplicable(sortSpecs)) {
  return new SimpleMerger(inSchema, left, right, comparator);
}
return new PairWiseMerger(inSchema, left, right, comparator);
'''